interview-questions.md

```diff
@@ -7,46 +7,51 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. In a model called Animal that has_many Sightings, what is the name of the foreign key? Would the foreign key be part of the Animal model or the Sightings model?
 
-  Your answer:
+  Your answer: The name of the foreign key would be the id of the matched Animal row. The foreign key would be apart of the Sightings model. 
 
-  Researched answer:
+  Researched answer: After looking at the syllabus it looks like a better answer to the first part of this question would be the foreign keys name would be animal_id:
```

Comment:
Your researched answer got it right! Your original answer was good too, but the researched answer provides a bit more what the interviewer would be looking for. Nice!